src/lib/serviceRoutines/postContextEntitiesByEntityId.cpp

```diff
@@ -41,6 +40,10 @@
 * postContextEntitiesByEntityId - 
 *
 * POST /ngsi9/contextEntities/{entityId}
```

Comment:
Should we globally change /ngsi9 -> /v1/registry and /ngsi10 -> /v1 in all documentation tokens like this one?